app/Livewire/Installer/Steps/QueueStep.php

```diff
@@ -48,7 +48,7 @@ public static function make(PanelInstaller $installer): Step
                     ->label(new HtmlString(trans('installer.queue.fields.crontab')))
                     ->disabled()
                     ->hintCopy()
-                    ->default('(crontab -l -u www-data 2>/dev/null; echo "* * * * * php ' . base_path() . '/artisan schedule:run >> /dev/null 2>&1") | crontab -u www-data -')
+                    ->default('(sudo crontab -l -u www-data 2>/dev/null; echo "* * * * * php ' . base_path() . '/artisan schedule:run >> /dev/null 2>&1") | sudo crontab -u www-data -')
```

Comment:
sudo might not always be installed.

Debian for example doesn't have it installed by default